Extensions/AzureAdAuthenticationBuilderExtensions.cs

```diff
@@ -19,28 +21,41 @@ public static AuthenticationBuilder AddAzureAd(this AuthenticationBuilder builde
             return builder;
         }
 
-        private class ConfigureAzureOptions: IConfigureNamedOptions<OpenIdConnectOptions>
+        private class ConfigureAzureOptions : IConfigureNamedOptions<OpenIdConnectOptions>
         {
             private readonly AzureAdOptions _azureOptions;
+            private readonly ITokenAcquisition _tokenAcquisition;
 
-            public ConfigureAzureOptions(IOptions<AzureAdOptions> azureOptions)
+            public ConfigureAzureOptions(IOptions<AzureAdOptions> azureOptions, ITokenAcquisition tokenAcquisition)
             {
                 _azureOptions = azureOptions.Value;
+                _tokenAcquisition = tokenAcquisition;
             }
 
             public void Configure(string name, OpenIdConnectOptions options)
             {
                 options.ClientId = _azureOptions.ClientId;
-                options.Authority = $"{_azureOptions.Instance}common/v2.0";   // V2 specific
+                options.Authority = $"{_azureOptions.Instance}{_azureOptions.TenantId}/v2.0";   // V2 specific
```

Comment:
null check on paramaters

Reply:
I'm not sure that a null check is necessary here since the Configure() method is never directly called in the code.